plugins/woocommerce/src/Internal/EmailEditor/WCTransactionalEmails/WCEmailTemplateGenerator.php

```diff
@@ -80,15 +165,19 @@ public function generate_email_templates() {
 		}
 	}
 
+	/**
+	 * Generate a single email template.
+	 *
+	 * This function generates a single email template post and sets its postmeta association.
+	 *
+	 * @param string $email_type The email type.
+	 * @param array $template_data The template data.
+	 * @return int The post ID of the generated template.
+	 */
 	private function generate_single_template($email_type, $template_data) {
-		// Skip if template already exists
-		if ($this->template_manager->template_exists($email_type)) {
-			return;
-		}
-
 		$post_data = [
 			'post_type' => Integration::EMAIL_POST_TYPE,
-			'post_status' => 'publish',
+			'post_status' => $template_data['enabled'] ? 'publish' : 'draft',
```

Comment:
I see! That's why the `Order details` aka `customer_invoice ` email is a draft. This one is manually sent so it will never be set as enabled. You can test if the email is manual by calling `WC_Email::is_manual` then perhaps the logic could be `$template_data['enabled'] || $template_data['manual'] ? 'publish' : 'draft'`

Reply:
Noted. Will do.

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/56567/commits/d9cf3bd20d4013f88a4b52766364afe876311c43